Detectors/MUON/MID/Workflow/src/RawDecoderSpec.cxx

```diff
@@ -39,9 +39,11 @@ class RawDecoderDeviceDPL
   void init(of::InitContext& ic)
   {
     auto stop = [this]() {
-      LOG(INFO) << "Capacities: ROFRecords: " << mDecoder->getROFRecords().capacity() << "  LocalBoards: " << mDecoder->getData().capacity();
-      double scaleFactor = 1.e6 / mNROFs;
-      LOG(INFO) << "Processing time / " << mNROFs << " ROFs: full: " << mTimer.count() * scaleFactor << " us  decoding: " << mTimerAlgo.count() * scaleFactor << " us";
+      if (mDecoder) {
+        LOG(INFO) << "Capacities: ROFRecords: " << mDecoder->getROFRecords().capacity() << "  LocalBoards: " << mDecoder->getData().capacity();
+        double scaleFactor = 1.e6 / mNROFs;
```

Comment:
@dstocco could you add a protection against division by `mNROFs=0`, then will merge w/o further tests, since the change is trivial.

Reply:
Hi @shahor02. Done. You can squash the commits when merging. Thanks!